backend/package.json

```diff
@@ -19,6 +19,7 @@
     "http": "^0.0.1-security",
     "https": "^1.0.0",
     "jsonwebtoken": "^8.5.1",
+    "loader": "^2.1.1",
```

Comment:
what is loader used for?

Reply:
When I ran the program it gave MODULE_NOT_FOUND error. I installed that module. But now it seems unnecessary. We can discard this line.